week-2/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,13 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  var first =  arr[0]
+  return first
 }
 
 function last(arr) {
-  return; // complete this statement
+  var lastValue = arr[arr.length-1]
```

Comment:
Is there a way to shorten this by not having a variable? Sometimes variables can help readability – does it help in this case?

Reply:
Very useful feedback! Thanks.